ci/install_yq.sh

```diff
@@ -28,6 +28,7 @@ function install_yq() {
 		local yq_path="${GOPATH}/bin/yq"
 	else
 		yq_path="/usr/local/bin/yq"
+                [ -x  "${yq_path}" ] && [ "`${yq_path} --version`"X == "yq (https://github.com/mikefarah/yq/) version ${yq_version}"X ] && return
```

Comment:
I see that this duplicates code from line 42 - could you refactor this so we don't duplicate this?

Reply:
Commit message updated and code refactored